src/test/java/htmlFindElements.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class htmlFindElements {
+    WebDriver driver;
+    private static final String Mysite = "C:\\Users\\Бубята\\IdeaProjects\\HerokuApp\\src\\test\\resources\\Test.html";
```

Comment:
Name for constants should be in UPPER case: MYSITE or MY_SITE

Reply:
done